PageObjectPattern/src/test/resources/features/apitest/spotifyCategories.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Api GET request to v1/browse/categories endpoint
```

Comment:
What about "Api GET request to v1/browse/categories endpoint" -> "Integration tests for v1/browse/categories endpoint"?